pyrobuf/parse_proto.py

```diff
@@ -252,25 +234,34 @@ def parse(self, cython_info=True, fname='', includes=None):
         return rep
 
     @classmethod
-    def parse_from_filename(cls, fname, includes):
+    def parse_from_filename(
+            cls,
+            fname,
+            includes,
+            disabled_tokens=tokens_not_supported_by_c
```

Comment:
Think this needs to be `self.tokens_not_supported_by_c`?

Reply:
Since it is within the function definition (the args) it is considered to be the still in the class's namespace. Nor the `self` nor `cls` are needed in this level. And unittests pass that way anyway ;)